packages/http-specs/specs/payload/content-negotiation/main.tsp

```diff
@@ -0,0 +1,61 @@
+import "@typespec/http";
+import "@typespec/spec-lib";
+
+using Http;
+using SpecLib;
+
+@doc("Test describing optionality of the request body.")
+@SpecLib.scenarioService("/content-negotiation")
+namespace Payload.ContentNegotiation;
+
+@SpecLib.scenario
```

Comment:
This can just be `@scenario` now that there is `using SpecLib;`

Reply:
@timotheeguerin I have removed it. It was working fine without the qualifier in the command line. But, in the VS Code editor it was giving a message as:

```
"scenarioService" is an ambiguous name between global.scenarioService, TypeSpec.SpecLib.scenarioService. Try using fully qualified name instead: global.scenarioService, TypeSpec.SpecLib.scenarioServiceTypeSpec(ambiguous-symbol)
```

Reply:
It shouldn't be, make sure you are not exporting things twice

Reply:
And have a clean build